react/src/components/molecules/EmergencyHeader/style.scss

```diff
@@ -0,0 +1,62 @@
+@import 'global';
+@import '02-molecules/emergency-header';
+
+@mixin ma-emergency-header ($color: $c-highlight) {
+  @if $color == $c-highlight {
+    color: $c-font-base;
+
+  } @else {
+    color: $c-font-inverse;
+  }
+  background-color: rgba($color, 1);
+  border-color: rgba($color, 1);
+  box-shadow: 0 0.25rem 0.5rem rgba(#000, 0.25);
```

Comment:
I don't see a `box-shadow` on the patternlab side. I agree that it should have one but maybe should be added on the organism instead, including the alerts not on top of alerts
```suggestion
```

Reply:
or if we are going to use this directly in budget, without the organism, then there should be a prop? what do you think @avrilpearl

Reply:
i think the organism as the show updates/emergency alert molecules are dependent on whether their are alerts or not